source/Calamari.Shared/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -570,6 +570,23 @@ function Import-CalamariModules() {
 	}
 }
 
+function ConvertTo-QuotedString([string]$arg){

```

Comment:
Given we added this to the root bootstrapper do we need to remove the same functions from Octopus.Features.WindowsService_BeforePostDeploy.ps1?